pokemongo_bot/cell_workers/update_title_stats.py

```diff
@@ -108,8 +107,10 @@ def _update_title(self, title, platform):
         :raise: RuntimeError: When the given platform isn't supported.
         """
         if platform == "linux" or platform == "linux2"\
-                or platform == "darwin" or platform == "cygwin":
+                or platform == "cygwin":
             stdout.write("\x1b]2;{}\x07".format(title))
+        elif platform == "darwin":
```

Comment:
The intended behavior seems backwards to me. Darwin is mac, right? So that would mean that iterm2 is cygwin, right? In which case, shouldn't the new format option be for cygwin instead of darwin?